index.js

```diff
@@ -20,3 +20,30 @@ module.exports = fn => {
 
 	return ret;
 };
+
+const log = (fn, promise) => {
+	console.log(`Promise from ${fn.displayName} resolved in ${promise.time} ms`);
```

Comment:
Weird. I ran it in Chrome DevTools. `displayName` is non-standard though. I guess the best would be to just do `fn.displayName || fn.name || '[anonymous]'`.